composer.json

```diff
@@ -9,26 +9,19 @@
             "email": "info@zicht.nl"
         }
     ],
-    "repositories": [
-        {
-            "type": "vcs",
-            "url": "git@github.com:zicht/classes.git"
-        }
-    ],
     "require": {
-        "php": "^7"
+        "php": "^7.2 || ^8.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4"
+        "phpunit/phpunit": "^8"
     },
     "autoload": {
-        "psr-4": { "": ["src/"] }
+        "psr-4": { "Zicht\\": ["src/Zicht"] }
```

Comment:
Voor alle classes waarvan de namespace begint met `Zicht\` zal de Composer autoloader gaan proberen te kijken in deze directory. In dit specifieke geval zou je ook kunnen kijken naar classmap i.p.v. psr-4 of misschien dat je met PSR-4 ook deze specifieke class name kunt targetten